### PR TITLE
Fix mixed indent in tf.py

### PR DESCRIPTION
**Pull request by klecki**
Applied autofomat using yapf

Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

#### Why we need this PR?
Fixes indentation and formatting in tf.py file.

#### What happened in this PR?
 - What solution was applied:
     Auto format with yapf.
 - Affected modules and functionalities:
     tf.py
 - Key points relevant for the review:
     N/a
 - Validation and testing:
     CI
 - Documentation (including examples):
     No

**JIRA TASK**: *[Use DALI-XXXX or NA]*

### CODE REVIEW

**Comment by klecki** [comment]
!build

**Review by mzient — APPROVED** [review]
Much as I don't like autoformatters, this one actually did a great job on this file.

File: dali/python/nvidia/dali/plugin/tf.py [hunk 1]
```diff
@@ -31,261 +32,285 @@
...
+
+    if serialized_pipeline is None:
+        serialized_pipeline = serialize_pipeline(pipeline)
+
+    # if batch_size is not provided we need to extract if from the shape arg
+    if (not isinstance(shapes, Iterable) or len(shapes) == 0) and batch_size == -1:
+        raise Exception(
+            'shapes and batch_size arguments cannot be empty, '
+            'please provide at leas one shape argument element with the BATCH size or set batch_size'
+        )
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/tf.py:77]: You could break the string differently, so the formatter would put the `)` in the line 76

File: dali/python/nvidia/dali/plugin/tf.py [hunk 2]
```diff
@@ -31,261 +32,285 @@
...
+                     dtypes=None,
+                     shapes=None):
+
+            output_shapes = self._handle_deprecation(output_shapes, shapes, "shapes")
+            output_dtypes = self._handle_deprecation(output_dtypes, dtypes, "dtypes")
+
+            if not self._check_output_dtypes(output_dtypes):
+                raise TypeError(("`output_dtypes` should be provided as single tf.DType value " +
+                    "or a tuple of tf.DType values. Got value `{}` of type `{}`.") \
+                        .format(output_dtypes, type(output_dtypes)))
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/tf.py:190]: nitpick-mark

File: dali/python/nvidia/dali/plugin/tf.py [hunk 3]
```diff
@@ -31,261 +32,285 @@
...
+            else:
+                return False
+
+        def _handle_deprecation(self, supported_arg, deprecated_arg, name):
+            if deprecated_arg is not None:
+                if supported_arg is not None:
+                    raise ValueError((
+                        "Usage of `{name}` is deprecated in favor of `output_{name}`. " +
+                        "Both arguments were provided, but only `output_{name}` should be provided."
+                    ).format(name=name))
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/tf.py:241]: nitpick-mark

File: dali/python/nvidia/dali/plugin/tf.py [hunk 4]
```diff
@@ -31,261 +32,285 @@
...
+                # show only this warning
+                warnings.warn(("Use of argument `{name}` is deprecated. Please use `output_{name}` instead. " \
+                    + "`output_{name}` should be provided as a tuple or a single value.").format(name=name),
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/tf.py:244]: nitpick-mark

File: dali/python/nvidia/dali/plugin/tf.py [hunk 5]
```diff
@@ -31,261 +32,285 @@
...
+                     device_id=0,
+                     exec_separated=False,
+                     prefetch_queue_depth=2,
+                     cpu_prefetch_queue_depth=2,
+                     gpu_prefetch_queue_depth=2,
+                     dtypes=None,
+                     shapes=None):
+            raise RuntimeError(
+                'DALIDataset is not supported for detected version of TensorFlow.  DALIDataset supports versions: 1.15, 2.0'
+            )
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/tf.py:310]: nitpick-mark

**Review by szalpal — APPROVED** [review]
Me gusta for the autoformatter :)

You could massage the parts, where the strings are strangely joined together due to previous formatter absence, but that's a nitpicking ;)

**Pull request merged**